### PR TITLE
Improve code coverage

### LINKED ISSUE #91: Revert tests from using assertNotEquals() back to assertFalse(object.equals(...))
JUnit's `assertNotEquals()` will not call `equals()` in the case where the first parameter is `null`, causing the instanceof block in `equals()` to never be entered and reducing code coverage. Let's revert those lines back to using `assertFalse(object.equals(...))` instead.

### PR DESCRIPTION
**Pull request by aureliony**
Closes #91

### CODE REVIEW

**Review by bryanyee33 — CHANGES_REQUESTED** [review]
LGTM, just a few nits.

File: src/test/java/seedu/address/model/person/fields/TagsTest.java [hunk 1]
```diff
@@ -27,31 +27,35 @@ public void constructor_emptyArray_success() {
...
         Tags expectedTags = new Tags(new Tag(VALID_TAG_1), new Tag(VALID_TAG_2));
 
         assertEquals(expectedTags, actualTags);
+        assertEquals(expectedTags.hashCode(), actualTags.hashCode());
     }
 
     @Test
     void equals_null_false() {
-        assertNotEquals(null, new Tags(emptyTagArray));
+        assertFalse(new Tags(emptyTagArray).equals(null));
```
[the remark this answers is not in the document]
Comment (aureliony) [reply src/test/java/seedu/address/model/person/fields/TagsTest.java:58]: Not required as all lines in `Tags::equals()` are already covered

[the remark this answers is not in the document]
Comment (bryanyee33) [reply src/test/java/seedu/address/model/person/fields/TagsTest.java:58]: Noted.

**Review by bryanyee33 — APPROVED** [review]
LGTM.

**Pull request merged**